src/core/agent/services/connectionService.test.ts

```diff
@@ -371,23 +372,35 @@ describe("Connection service of agent", () => {
   test("can delete conenction by id", async () => {
     Agent.agent.getKeriaOnlineStatus = jest.fn().mockReturnValueOnce(true);
     connectionNoteStorage.findAllByQuery = jest.fn().mockReturnValue([]);
+    getIpexMessageMetadataByConnectionIdMock.mockResolvedValueOnce([]);
     const connectionId = "connectionId";
     await connectionService.deleteConnectionById(connectionId);
     expect(connectionStorage.deleteById).toBeCalledWith(connectionId);
     // expect(deleteContactMock).toBeCalledWith(connectionId); // it should be uncommented later when deleting on KERIA is re-enabled
   });
 
-  test("Should delete connection's notes when deleting that connection", async () => {
+  test("Should delete connection's notes & history when deleting that connection", async () => {
     Agent.agent.getKeriaOnlineStatus = jest.fn().mockReturnValueOnce(true);
     connectionNoteStorage.findAllByQuery = jest.fn().mockReturnValue([
       {
         id: "uuid",
         title: "title",
       },
     ]);
+    getIpexMessageMetadataByConnectionIdMock.mockResolvedValueOnce([
+      {
+        id: "id",
+        credentialType: "rare evo",
+        content: {},
+        historyType: ConnectionHistoryType.CREDENTIAL_ISSUANCE,
+        createdAt: new Date(),
+        connectionId: "connectionId",
+      },
+    ]);
     const connectionId = "connectionId";
     await connectionService.deleteConnectionById(connectionId);
     expect(connectionNoteStorage.deleteById).toBeCalledTimes(1);
+    expect(ipexMessageStorage.deleteIpexMessageMetadata).toBeCalledTimes(1);
```

Comment:
for this line and the line above, can we use `toBeCalledWith()` to check the ID we are deleting